package.json

```diff
@@ -2,7 +2,8 @@
   "name": "react-syntax-highlighter",
   "version": "9.0.0",
   "description": "syntax highlighting component for react with prismjs or highlightjs ast using inline styles",
-  "main": "index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
```

Comment:
This change is required to get SSR to work. For the server side renderer variant we want to use the commonjs variant but for web we always want to use esm (import/export for tree shaking and dynamic import for code splitting)